caffe2/python/__init__.py

```diff
@@ -0,0 +1,12 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
+from caffe2.proto import caffe2_pb2
+# TODO: refactor & remove the following alias
```

Comment:
Nice to have a little comment here saying what this is all about, given that we had so much discussion about how to do this :)